### DISCUSSION TITLE
Change backgound color for Navigation

### DISCUSSION THREAD

**Discussion by MatLish00010** [expo/router#631] [Q&A]
Hello! I need to use a background image for my app and I tried using

`<Background>
        <Slot />
      </Background>
`
in root but it doesn't work because navigation has background color, how can i remove bgc for expo-router?

**Comment by kadikraman** [comment]
You can set `backgroundColor: 'transparent'` to remove the background color.

So e.g. in a Stack:
```tsx
<Stack screenOptions={{ contentStyle: { backgroundColor: 'transparent'} }}>
```

Or if you're using `ThemeProvider`, it'll be something like:
```tsx
<ThemeProvider value={{
  ...,
  colors: {
    ...,
     background: 'transparent',
  },
}}>
```

**Reply by bim-oulabi** [reply]
When using stack, its difficult to set a custom background component with transparent content style since the transition will look weird

```
<AuthGradient>
        <Stack
          screenOptions={{
            headerShown: false,
            // animation: 'slide_from_right',
            contentStyle: {
              backgroundColor: 'transparent',
            },
          }}>
          <Stack.Screen name='login' />
          <Stack.Screen name='signup' />
          <Stack.Screen name='forgot-password' />

     </Stack>
      </AuthGradient>
```
https://github.com/expo/router/assets/135221263/9730867a-6105-416f-aa12-6d181ed2e63b

**Reply by SunZhiC** [reply]
> You can set `backgroundColor: 'transparent'` to remove the background color.
> 
> So e.g. in a Stack:
> 
> ```tsx
> <Stack screenOptions={{ contentStyle: { backgroundColor: 'transparent'} }}>
> ```
> 
> Or if you're using `ThemeProvider`, it'll be something like:
> 
> ```tsx
> <ThemeProvider value={{
>   ...,
>   colors: {
>     ...,
>      background: 'transparent',
>   },
> }}>
> ```

Below code doesn't work for me, if I change it to 'green', it works well, but not works for 'transparent', I suppose there is a under layer control the `backgroundColor`, I am looking for a work around.
```
<Stack screenOptions={{ contentStyle: { backgroundColor: 'transparent'} }}>
```

**Comment by M-Ali-Haider** [comment]
Is there something like this in the Tabs?
